tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "noImplicitAny": false,
         "noEmit": true,
         "target": "ES2020",
-        "module": "ES2020",
+        "module": "Node16",
```

Comment:
I wonder if we can restrict this to the only the directory where we need this? Most of the code can not use the node api ...

Reply:
I'm trying to think of why I made this change ...

Reply:
I was getting this [error](https://github.com/vector-im/hydrogen-web/runs/7423907806?check_suite_focus=true) but I no longer get it when I revert this change.

Reply:
Reverted in https://github.com/vector-im/hydrogen-web/pull/769/commits/39817dc36bb8952aa7ce55045c7ead6f506c776f